src/FitterBase.cc

```diff
@@ -190,7 +192,12 @@ MinimizeResult FitterBase::minimize(std::string const &whatToFit, double nSigmaC
         grad *= -1;
     }
 
-    LOGLS_INFO(_log, "Total number of outliers " << totalOutliers);
+    // only print the outlier summary if outlier rejection was turned on.
+    if (nSigmaCut != 0) {
+        LOGLS_INFO(_log, "Total number of outliers (Measured + Reference = Total): "
+                                 << totalMeasOutliers << " + " << totalRefOutliers << " = "
+                                 << totalMeasOutliers + totalRefOutliers);
```

Comment:
The wording "Total of X + Y = Total" seems a bit weird. Is the initial "Total" still meaningful?